src/components/calcite-value-list/calcite-value-list.tsx

```diff
@@ -199,6 +199,56 @@ export class CalciteValueList {
 
   getItemData = getItemData.bind(this);
 
+  keyDownHandler = (event) => {
+    const handleElement = event.composedPath().find((item) => {
+      return item.dataset?.jsHandle;
+    });
+    const valueListElement = event.composedPath().find((item) => {
```

Comment:
nitpick: maybe this could be a method in the dom utility?

Reply:
possibly, the inner function may need to be unique. If we start seeing more of a pattern arise then we could abstract this.